handlers/stripe-disputes/src/handlers/disputeClosedHandler.ts

```
@@ -0,0 +1,89 @@
+import type { Logger } from '@modules/logger';
+import { getIfDefined } from '@modules/nullAndUndefined';
+import { stageFromEnvironment } from '@modules/stage';
+import { ZuoraClient } from '@modules/zuora/zuoraClient';
+import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { listenDisputeClosedInputSchema } from '../dtos';
+import type { ZuoraInvoiceFromStripeChargeIdResult } from '../interfaces';
+import {
+	upsertSalesforceObject,
+	zuoraGetInvoiceFromStripeChargeId,
+} from '../services';
+import type { SalesforceUpsertResponse } from '../types';
+
+/**
+ * Creates a handler function for processing Stripe dispute closed webhooks
+ *
+ * @param logger - Logger instance for tracking webhook processing
+ * @returns Handler function that processes API Gateway events for dispute closure
+ */
+export function listenDisputeClosedHandler(logger: Logger) {
```

This might be a requirement to look at in a future seperate PR, but we must do this processing _asynchronously_ to avoid causing problems with the Stripe webhook notifications. 

Essentially we need to return a `200` to Stripe as quickly as possible. Remember Stripe itself cannot do anything with an error, except retry it again, so for any problems we need to throw an error and have a Cloudwatch alarm to resolve internally.

Zuora can be notoriously slow to respond, particularly for things like subscription cancellations which in extreme circumstances can take 30-60 seconds or longer. So it would not be a good idea to try and do all the operations BEFORE sending the response back to Stripe.

Just mentioning now in case it's easier to incorporate this now, but fine to cover later!